apis/configuration/slo_controller_config.go

```diff
@@ -295,6 +297,32 @@ type ModelRatioCfg struct {
 	HyperThreadTurboEnabledRatio *float64 `json:"hyperThreadTurboEnabledRatio,omitempty"`
 }
 
+// ResourceAmplificationCfg is the cluster-level configuration of the resource amplification strategy.
+// +k8s:deepcopy-gen=true
+type ResourceAmplificationCfg struct {
+	ResourceAmplificationStrategy `json:",inline"`
+	NodeConfigs                   []NodeResourceAmplificationCfg `json:"nodeConfigs,omitempty" validate:"dive"`
+}
+
+// NodeResourceAmplificationCfg is the node-level configuration of the resource amplification strategy.
+// +k8s:deepcopy-gen=true
+type NodeResourceAmplificationCfg struct {
+	NodeCfgProfile `json:",inline"`
+	ResourceAmplificationStrategy
+}
+
+// ResourceAmplificationStrategy is the resource amplification strategy.
+// +k8s:deepcopy-gen=true
+type ResourceAmplificationStrategy struct {
+	// Enable defines whether the resource amplification strategy is enabled.
+	Enable *bool `json:"enable,omitempty"`
+	// CPURatio defines the cpu amplification ratio.
+	CPURatio *float64 `json:"CPURatio,omitempty"`
+	// MemoryRatio defines the memory amplification ratio.
+	MemoryRatio                *float64                        `json:"memoryRatio,omitempty"`
+	ResourceAmplificationRatio map[corev1.ResourceName]float64 `json:"resourceAmplificationRatio,omitempty"`
```

Comment:
As we have this field, `CPURatio` and `MemoryRatio` can be removed. Also don't forget to add comment for this field.